release.sh

```diff
@@ -98,6 +103,29 @@ commit_belongs_to_release_branch() {
     return 1
 }
 
+new_snapshot () {
+    local sha latest latest_prefix prefix new_line tmp
+    sha=$(git rev-parse HEAD)
+    if ! commit_belongs_to_release_branch $sha; then
+        echo "WARNING: Commit does not belong to a release branch."
+    fi
+    latest=$(head -1 LATEST | awk '{print $2}')
+    latest_prefix=$(echo $latest | grep -o -P "$STABLE_REGEX" | head -1)
+    if is_stable $latest; then
+        prefix=$(echo $latest_prefix | jq -Rr '. | split(".") | [.[0], (.[1] | tonumber + 1 | tostring), "0"] | join(".")')
+    else
+        prefix=$latest_prefix
+    fi
+    new_line=$(make_snapshot $sha $prefix)
+    tmp=$(mktemp)
```

Comment:
Do you need to clean this up?

Reply:
I don't think so, why? The reason for using a tmpfile here is that we're going to (conceptually) pipe from LATEST to LATEST and Bash tends to truncate before it reads on those cases.